packages/common/src/feedback/_internal/FeedbackSchema.ts

```diff
@@ -0,0 +1,23 @@
+import { IConfigurationSchema } from "../../core";
+import { HubItemEntitySchema } from "../../core/schemas/shared/HubItemEntitySchema";
+
+export type FeedbackEditorType = (typeof FeedbackEditorTypes)[number];
+export const FeedbackEditorTypes = [
+  "hub:feedback:edit",
+  "hub:feedback:settings",
+] as const;
+
+/**
+ * defines the JSON schema for a Feedback entity's editable fields
+ */
+export const FeedbackSchema: IConfigurationSchema = {
+  ...HubItemEntitySchema,
+  properties: {
+    displayMap: {
```

Comment:
This property should be named whatever we settle on in my [comment](https://github.com/Esri/hub.js/pull/1414/files#r1491448517) above per the second bullet point. Again, open to suggestions on naming and/or values. Options are:

`displayMap: boolean`, `displayMode: "standard" | "map"`, `isMapFeedback: boolean`